src/ucp/api/ucp.h

```diff
@@ -2634,6 +2634,15 @@ ucs_status_ptr_t ucp_tag_send_sync_nb(ucp_ep_h ep, const void *buffer, size_t co
  *
  * @note The user should not modify any part of the @a buffer after this
  *       operation is called, until the operation completes.
+ * @note The argument @a param->op_attr_mask should be OR combination of
+ *       flags UCP_OP_ATTR_FIELD_REQUEST, UCP_OP_ATTR_FIELD_CALLBACK,
+ *       UCP_OP_ATTR_FIELD_DATATYPE, UCP_OP_ATTR_FLAG_NO_IMM_CMPL,
+ *       UCP_OP_ATTR_FLAG_NO_ZCOPY and UCP_OP_ATTR_FLAG_FORCE_IMM_CMPL.
+ * @note Combination of flags
+ *       UCP_OP_ATTR_FLAG_NO_IMM_CMPL | UCP_OP_ATTR_FLAG_FORCE_IMM_CMPL is not
```

Comment:
this is a general note for any function. Please describe this once where the flags are defined